frontend/src/util/analytics.ts

```diff
@@ -93,11 +94,21 @@ const page = (name: string, properties?: rudderanalytics.apiObject) => {
 	rudderanalytics.page(name, omit(properties, rudderstackReserved))
 }
 
+const trackGaEvent = (event: string, properties?: Record<string, any>) => {
+	if (window.dataLayer) {
+		window.dataLayer.push({
+			event,
+			...properties,
+		})
+	}
+}
```

Comment:
should we just add this to the `track` function and have all track events reported as GA events? or is that too noisy

Reply:
Yeah, I wasn't sure about that. I remember being much more strict about sending new events into GA vs product analytics tools, so I'm inclined not to do that yet, but maybe something we should ask GR0 about. I added it to a doc I'll reference for our chats with them here: https://docs.google.com/document/d/1F2-kB-HScsfgEVe9Ou-PlEc_szGcdReRocOKecHWlGU/edit?usp=sharing